clustered/client/src/main/java/org/ehcache/clustered/client/config/builders/ClusteringServiceConfigurationBuilder.java

```diff
@@ -128,7 +135,7 @@ public ClusteringServiceConfigurationBuilder readOperationTimeout(long duration,
 
   @Override
   public ClusteringServiceConfiguration build() {
-    return new ClusteringServiceConfiguration(clusterUri, timeouts, autoCreate, null);
```

Comment:
If you want to extract out the commonality here I would just have this method call `build(null)`

Reply:
Done